### PR TITLE
add meta-coral to ignore list

### PR DESCRIPTION
**Pull request by AsuraZeng**
ignore the meta-coral folder

Signed-off-by: chao zeng <chao.zeng@siemens.com>

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,4 @@ isar/
 /cip-core
 recipes-core/customizations-base/local.inc
 .config.yaml
+meta-coral/
```
Comment (jan-kiszka) [inline .gitignore:6]: `/meta-coral` - yeah, needed.

Comment (jan-kiszka) [reply .gitignore:6]: And maybe sort this to the other layers.

**Review by jan-kiszka — APPROVED** [review]
You didn't follow my other suggestion, but it's still fine.

**Comment by AsuraZeng** [comment]
> You didn't follow my other suggestion, but it's still fine.
done the change. pls help to check.

**Comment by jan-kiszka** [comment]
Yep, that's what I had in mind.

**Pull request merged**